source/common/upstream/cds_api_impl.h

```diff
@@ -49,7 +52,7 @@ class CdsApiImpl : public CdsApi,
 
   ClusterManager& cm_;
   std::unique_ptr<Config::Subscription<envoy::api::v2::Cluster>> subscription_;
-  std::string version_info_;
+  std::string whole_update_version_info_;
```

Comment:
Prefer to stick with system/resource version for naming internally as well.

Reply:
Unfortunately this is necessary due to the mismatch between vanilla xDS and incremental xDS's approaches to version. In vanilla, these xDS-container-protobuf-level versions are *the* versions. In incremental, they're indicated as basically just debug info. We can accommodate both of those meanings with this single variable, but it needs a name that vaguely describes both.

Reply:
Hmm, I think we can consider the existing use in xDS as system version still, since it's a logical clock and doesn't imply resource level version (despite how we use it today).

Reply:
:+1: done.